### PR TITLE
Refs #12970 - ejs 1.1.1 for Patternfly sprockets

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: rubygem-ejs/rubygem-ejs.spec [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+%global gem_name ejs
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.1.1
+Release: 3%{?dist}
+Summary: EJS (Embedded JavaScript) template compiler
+Group: Development/Languages
+License: MIT
+URL: https://github.com/sstephenson/ruby-ejs/
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
```
Comment (domcleal) [inline rubygem-ejs/rubygem-ejs.spec]: Please change to HTTP so git-annex on EL6 can fetch it.

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -229,6 +229,7 @@
       <packagereq type="default">tfm-rubygem-clamp-doc</packagereq>
       <packagereq type="default">tfm-rubygem-daemons-doc</packagereq>
       <packagereq type="default">tfm-rubygem-deep_cloneable-doc</packagereq>
+      <packagereq type="default">tfm-rubygem-ejs-doc</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-rhel7.xml]: missing main tfm-rubygem-ejs package

**Comment by dLobatog** [comment]
Latest update adds tfm-rubygem-ejs to comps on el6 and changed the https rubygems source so that el6 can fetch it

**Comment by domcleal** [comment]
Thanks @dLobatog, merged and building.

**Pull request merged**